src/pq_flash_index.cpp

```diff
@@ -717,24 +776,24 @@ int PQFlashIndex<T, LabelT>::load_from_separate_paths(uint32_t num_threads, cons
             assert(medoid_stream.is_open());
             std::string line, token;
 
-            _filter_to_medoid_id.clear();
+            _filter_to_medoid_ids.clear();
             try
             {
                 while (std::getline(medoid_stream, line))
                 {
                     std::istringstream iss(line);
                     uint32_t cnt = 0;
-                    uint32_t medoid = 0;
+                    std::vector<uint32_t> medoids;
                     LabelT label;
                     while (std::getline(iss, token, ','))
                     {
                         if (cnt == 0)
                             label = (LabelT)std::stoul(token);
                         else
-                            medoid = (uint32_t)stoul(token);
+                            medoids.push_back((uint32_t)stoul(token));
                         cnt++;
                     }
-                    _filter_to_medoid_id[label] = medoid;
+                    _filter_to_medoid_ids[label] = medoids;
```

Comment:
use "std::move(mediods)" to avoid copying vector elements

Reply:
done